api/dbadapters/redshift.ts

```diff
@@ -137,17 +139,41 @@ export class RedshiftDbAdapter implements IDbAdapter {
     }));
   }
 
+  public async search(searchText: string): Promise<dataform.ITableMetadata[]> {
+    const results = await this.execute(
+      `select tables.table_schema as table_schema, tables.table_name as table_name
+       from svv_tables as tables
+       left join svv_columns as columns on tables.table_schema = columns.table_schema and tables.table_name = columns.table_name
+       where tables.table_schema like $1 or tables.table_name like $1 or tables.remarks like $1
+         or columns.column_name like $1 or columns.remarks like $1
+       group by 1, 2`,
+      {
+        params: [`%${searchText}%`],
+        rowLimit: 100
```

Comment:
Feels like these limits could hurt us defaulting to 100, can we bump default up a bit? 1K?